src/utils/hub-channel.js

```diff
@@ -96,6 +96,14 @@ export default class HubChannel {
     this.channel.push("message", { body, type });
   };
 
+  pin = (id, gltfNode) => {
+    this.channel.push("pin", { id, gltf_node: gltfNode });
```

Comment:
so in order for persistent to outlast the last person who disconnects from janus, this could be caught with something in `hub_channel.ex` like
```
  def handle_in("pin", _payload, socket) do
    hub = socket |> hub_for_socket
    # do something to save the object in the payload for later use
    {:noreply, socket}
  end
```
is that right?

Reply:
yup! it's already in fact live :)

https://github.com/mozilla/reticulum/blob/master/lib/ret_web/channels/hub_channel.ex#L86

Reply:
https://github.com/mozilla/reticulum/pull/78

Reply:
note that PR includes the web push stuff too since i was working against that base branch

Reply:
Oh right, woops -- I was looking at reticulum from 100 commits ago O_o